### ISSUE TITLE
Localization: how to pass a <key> into a <key>

### ISSUE THREAD

**Issue by ildar170975** [thomasloven/lovelace-template-entity-row#82]
This code shows a localized "Home" or "Away" strings:
```
      - type: custom:template-entity-row
        entity: person.ildar
        name: localized
        state: _(component.device_tracker.state._.{{states(config.entity)}})
```
Here `component.device_tracker.state._.home` and `component.device_tracker.state._.not_home` keys are used.

Some keys have "internal" keys, like:
```
ui.errors.config.key_missing: "Required key ''{key}'' is missing.”
```
To show the whole message, I need to pass a "key" into it.

How should I combine the `_(ui.errors.config.key_missing)` and this "internal" key?

**Comment by ildar170975** [comment]
Potentially may be closed by this PR: https://github.com/thomasloven/lovelace-template-entity-row/pull/85

**Comment by thomasloven** [comment]
Fixed by #85
